packages/baseclient-components/src/container/Footer/Footer.tsx

```diff
@@ -22,13 +39,77 @@ interface FooterState {
  */
 export default class Footer extends React.Component<FooterProps, FooterState> {
 
+  private predefinedCrsDefinitions = [{
+    code: '4326',
+    value: 'WGS 84'
+  }, {
+    code: '3857',
+    value: 'Pseudo-Mercator -- Spherical Mercator'
+  }];
+
   /**
    * Create the Footer.
    *
    * @constructs Footer
    */
   constructor(props: FooterProps) {
     super(props);
+
+    this.setProjection = this.setProjection.bind(this);
+  }
+
+  /**
+   * Is invoked before a mounted component receives new props.
+   *
+   * @param {Object} nextProps The next props.
+   */
+  componentDidMount() {
+    const {
+      map
+    } = this.props;
+    if (map) {
+      this.createOlMousePositionControl(map);
+    }
+  }
+
+  /**
+   * Creates and adds the mouse position control to the map.
+   */
+  createOlMousePositionControl = (map: any) => {
+    const mousePositionControl = new OlMousePositionControl({
+      coordinateFormat: createStringXY(2),
+      projection: map.getView().getProjection().getCode(),
+      target: document.getElementById('mouse-position'),
+      undefinedHTML: '&nbsp;'
+    });
+
+    map.addControl(mousePositionControl);
```

Comment:
Please check if the `ol/control/MousePosition` has been added to the map already.